src/main/java/com/gbtec/Main.java

```diff
@@ -1,22 +1,59 @@
 package com.gbtec;
 
-import static com.gbtec.service.GenerationCalculator.calculateNextGeneration;
-import static com.gbtec.service.GenerationReader.createInitialGeneration;
-
-import com.gbtec.model.Generation;
-import com.gbtec.presentation.Printer;
 
 public class Main {
 
-    public static void main(String[] args) {
-        Printer printer = new Printer();
-        Generation currentGeneration = createInitialGeneration();
+    public static void main(String[] args) throws InterruptedException{
+        int sizeOfGeneration = 50;
+        int numberOfNextGenerations = 100;
+        int wishPattern = 1;
+
+        // Select which kind of pattern you want
+        Generation generation = new Generation(sizeOfGeneration);
+        if (wishPattern == 0) {
+            GenerationRandomizer.randomizeGeneration(generation, sizeOfGeneration);
```

Comment:
This code relates to the setup of a generation. You can also move this code in a method and name the methos so that it is more understandable like "setupRandomGeneration()"